src/sentry/grouping/enhancement-configs/common@2019-03-23.txt

```diff
@@ -61,3 +61,6 @@ family:native function:"?[[]SentryCrash *"                          -app -group
 family:native function:"?[[]SentryClient *"                         -app -group
 family:native function:"?[[]RNSentry *"                             -app -group
 family:native function:"?[[]SentrySDK *"                             -app -group
+
+# Java
+family:other module:*EnhancerByGuice*                          -app -group
```

Comment:
`m`: I think it would be nice to add a comment on what `EnhancerByGuice` actually is and which problem this rule solves. If I'm not mistaken it is generated by [Google Guice](https://github.com/google/guice/blob/fdfc81cde4f0dc1811489715a34b3632d9dfacd5/core/src/com/google/inject/internal/BytecodeGen.java#L69).

Reply:
Literally pasting this string shows up what this is about though, not sure:

# Generated code by Google Guice 

is as useful as it is noise.

Reply:
> Thanks, @bruno-garcia for taking care of this. I understand how this can solve #21645, but I don't get it how this solves #18959, as this PR doesn't include a rule about `GeneratedConstructorAccessorXX`.

This is a fair point, I'll add a rule for this too.

Reply:
I agree that you can find out in a minute were `EnhancerByGuice` comes from. Still, we have some information in the issues and also in this PR that is not obvious by this line of code. So you would have to do `git blame` to find the corresponding PR and read the issue to find out why this rule is here.